### DISCUSSION TITLE
Time and memory complexity in Past Solutions

### DISCUSSION THREAD

**Discussion by OwnYhide** [codewars/codewars.com#3087] [Ideas]
I tend to solve some problems multiple times using different approaches. Over time, I often revisit my solutions to review them. From my perspective, it would be nice to see the time and memory complexity of my previous solutions just to reconsider some things and enjoy my past skills. What do you think about it?

**Comment by hobovsky** [comment]
I think that this question is a duplicate and has been asked, and answered, a couple of times already :) For example in #959 and other tickets linked from there.
